docs/spec/api.md

```diff
@@ -618,26 +618,6 @@ The "digest" parameter must be included with the PUT request. Please see the
 [_Completed Upload_](#completed-upload) section for details on the parameters
 and expected responses.
 
-Additionally, the upload can be completed with a single `POST` request to
```

Comment:
@stevvooe cool - how do you guys update the docs for the release that's out there, and avoid releasing docs changes for vNext?

I'm happy to kick around ideas, what we're looking for in the end is something we can consistently apply to all repos with docs.